packages/backend-lib/src/server/genericErrorMiddleware.ts

```diff
@@ -19,11 +18,16 @@ export interface GenericErrorMiddlewareCfg {
   formatError?: (err: ErrorObject) => void
 }
 
+export interface ErrorReportingService {
+  // biome-ignore lint/suspicious/noExplicitAny: it is expected of the `captureException` to be tolerant of any argument
+  captureException: (err: any) => string | undefined
```

Comment:
p3: I'd clarify in jsdoc what is that `string` that is expected to be returned, and why it can be undefined (which is a trickier question, but we at least should try to answer the question)